src/webpack/loaders/devSiteEntry.js

```diff
@@ -0,0 +1,83 @@
+const lodashTemplate = require('lodash.template');
+const { getOptions } = require('loader-utils');
+
+const generateNavigationConfig = require('../loaderUtils/generateNavigationConfig');
+
+/**
+ * Generate the example with the supplied file.
+ * Don't use an arrow function
+ */
+const loader = async function loader(template) {
+  const callback = this.async();
+
+  const imports = {};
+  let index = 0;
+
+  const addImport = (path) => {
+    let ident = imports[path];
+    if (ident) {
+      return ident;
+    }
+    ident = `import${index}`;
+    index += 1;
+    imports[path] = ident;
+    return ident;
+  };
```

Comment:
Could this be static outside of the loader func?

Reply:
Originally it was, but then imports from other plugins started showing up in all plugins. So i had to make it instance based.